assets/javascript/nailed/index.js

```diff
@@ -1,14 +1,19 @@
 function index(colors, product_query, org_query){
-  $.getJSON("/json/github/donut/allpulls", function (json) {
+  $.getJSON("/json/changes/donut/allchanges", function (json) {
+    function set_col() {
+      var cols = new Array();
+      for ( var x in json) {
+        if (json[x]["origin"] == "github")
+          cols.push(colors["github"]["col".concat(parseInt(x)%5+1)]);
```

Comment:
I get a javascript error about `colors.github` being undefined which originates from this line. Where do you define the colors hash?

Reply:
Ahh, I see, you created new colors in the `colors.yml.exampe`, which I haven't added to my `colors.yml` yet.

Reply:
Maybe it would make sense to set some default colors, so it doesn't fail when none are set (the donuts don't render). But this does not have to be part of this PR.